packages/core-flows/src/file/steps/delete-files.ts

```diff
@@ -0,0 +1,17 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import { IFileModuleService } from "@medusajs/types"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+export const deleteFilesStepId = "delete-files"
+export const deleteFilesStep = createStep(
+  { name: deleteFilesStepId, noCompensation: true },
```

Comment:
you can remove `noCompensation` and by not defining the compensation function as the second argument, this flag will be true.